### DISCUSSION TITLE
Mevo+ Issues with set up.

### DISCUSSION THREAD

**Discussion by gerardkele** [springbok/MLM2PRO-GSPro-Connector#90] [General]
I am not able to get past this issue. Something about list index out of range.  Lock the program up.
![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/167823579/f45cdf2d-4b54-403c-959e-16732e7f3ea5)
 Also, do I need to get a different License for GSPro? All I need this to do is putting nothing else since my mevo works fine with GSPro other than putting. I am new to all this API connector stuff.

**Comment by Wallace7777** [comment]
I don’t think you can use the webcam putting separate with the Flightscope native software embedded in GSPRO. I think you have to use the connector for both the Mevo+ and the webcam putting as standalone. I wish we could just use the webcam putting separate from the embedded software, I think there is an issue with chipping using the connector… unless I am wrong and someone can correct me.

**Reply by RocknRollGolf** [reply]
No Issue with chipping, just use the play golf mode in FS Golf Pc, and switch between full shot and chipping.

**Reply by obijuanp** [reply]
you can use voice attack to set a keyboard hotkey. i use a streamdeck to then press that hotkey and switch. but you do have to switch it in FS for anything under 10 yards to read.

**Comment by gerardkele** [comment]
Ok then. Thanks for the help on that part. . Any advice on this error message ?  I think someone else posted about something similar. I would still like to get it set up and see if it works for putting. Posted videos look very interesting.

**Reply by obijuanp** [reply]
Dont use the + sign in the name it break its just name it mevo.

**Reply by gerardkele** [reply]
Ahhhh that makes sense. Will give it a shot.

**Reply by RocknRollGolf** [reply]
they need to up date the WIKI , Dont use the + sign at the end of Mevo

**Comment by RocknRollGolf** [comment]
they need to up date the WIKI , Dont use the + sign at the end of Mevo

**Reply by springbok** [reply]
I'll look at filtering out any invalid filename characters, or at least display an error message.

**Reply by Wrightn13** [reply]
is there a way to add more boxes in the roi? id like to see the club path and face to path that i get from the mevo plus but its not an option to put another box

**Comment by RocknRollGolf** [comment]
Also if you have the updated FS Golf App, 
Mirror App Window Name: 'FS GOLF PC 2.0' It's important to get this right otherwise the Connector won't be able to locate the correct window when taking a screenshot.
Dont use 2.0 at the end just use FS Golf 
PC

**Reply by gerardkele** [reply]
Yes , I found that when I used the look up open window button , it did not show 2.0 at the end of the name. Dropped it and worked perfect.
